interop-tests/src/test/scala/org/apache/pekko/grpc/scaladsl/GrpcExceptionDefaultHandleSpec.scala

```diff
@@ -58,7 +58,7 @@ class GrpcExceptionDefaultHandleSpec
             case Seq(LastChunk("", List(`Status`("3")))) => // ok
           }
         case _: Strict =>
-          response.attribute(AttributeKeys.trailer).get.headers.contains("grpc-status" -> "3")
+          response.headers.find(_.is("grpc-status")).map(_.value()) shouldBe Some("3")
```

Comment:
The spec at https://github.com/grpc/grpc/blob/master/doc/PROTOCOL-HTTP2.md says "Status must be sent in Trailers even if the status code is OK.". Perhaps we should allow both cases in this test? I.e. see if there is an `AttributeKeys.trailer`, if there is an `AttributeKeys.trailer` check that it has a grpc-status of 3 and the `response.headers` does *not* have a status, if there is no `AttributeKeys.trailer` then check the status in `response.headers`?

Reply:
Trying to understand what you mean, it is allowed under the spec but knowing the implementation of the exception handler I know it will be a Strict. So adding another match arm that is never hit won't do much. But we _**should**_ probably make sure that such a test exist that HTTP Responses produced should 100% have trailers in one of the two possible locaitons. I don't know if this is the test to do it in tho, wdyt?

Reply:
IMHO, a test should not describe "what the current implementation does", but "what the requirements on a correct implementation are": if you change the implementation from one 'correct' approach to another, ideally you shouldn't have to change the test. That said, if it makes the test too complex I'm OK with leaving it like this.

Reply:
That is a valid point, I agree. I will add the match arm here